opensoundscape/preprocess/actions.py

```diff
@@ -33,7 +33,7 @@ class BaseAction:
     """
 
     def __init__(self):
-        self.params = pd.Series(dtype=object)
```

Comment:
We should make this change throughout the codebase to be consistent if we need to.